pygfx/geometries/_base.py

```diff
@@ -84,24 +93,24 @@ def __init__(self, **kwargs):
             # Store
             setattr(self, name, resource)
 
-    def __setattr__(self, key, new_value):
+    def __setattr__(self, key: str, new_value: Resource) -> None:
         if not key.startswith(("_", "morph_")):
             if isinstance(new_value, Trackable) or key in self._store:
                 return setattr(self._store, key, new_value)
         object.__setattr__(self, key, new_value)
 
-    def __getattribute__(self, key):
+    def __getattribute__(self, key: str) -> Resource:
         if not key.startswith(("_", "morph_")):
             if key in self._store:
                 return getattr(self._store, key)
         return object.__getattribute__(self, key)
 
-    def __dir__(self):
+    def __dir__(self) -> Iterable[str]:
         x = object.__dir__(self)
         x.extend(dict.keys(self._store))
         return x
 
-    def get_bounding_box(self):
+    def get_bounding_box(self) -> npt.NDArray[np.float32] | None:
```

Comment:
If we `from numpy.typing import NDArray, ...` at the top, we can make this a bit more neat, I think?

```suggestion
    def get_bounding_box(self) ->NDArray[np.float32] | None:
```
Looking at the rendered docs:
<img width="490" alt="image" src="https://github.com/user-attachments/assets/9e5c2c7e-511d-4857-a5ba-101aeaabda9e" />

Reply:
done